AffiliateWP/AffiliateWP
# issue/985 – Mark $0.00 referrals in Formidable Pro as 'Unpaid'

Issue: #985

To test:
1. Create a form with two fields, the first to serve as a 'Description', the second to serve as the 'Amount'.
2. In the form settings, under the AffiliateWP tab, assign the two required fields for 'Description' and 'Amount', respectively
3. Create a new page and embed the form
4. Visit the new page on a valid referral URL
5. Enter a description and either leave the amount blank or enter 0.
6. Submit the form.

A $0.00 referral should get generated and should be marked 'unpaid'.
